src/command_modules/azure-cli-container/azure/cli/command_modules/container/custom.py

```diff
@@ -166,18 +174,22 @@ def create_container(cmd,
         volumes.append(gitrepo_volume)
         mounts.append(gitrepo_volume_mount)
 
-    cgroup_ip_address = _create_ip_address(ip_address, ports, protocol, dns_name_label)
-
     # Concatenate secure and standard environment variables
     if environment_variables and secure_environment_variables:
         environment_variables = environment_variables + secure_environment_variables
     else:
         environment_variables = environment_variables or secure_environment_variables
 
+    # Set up VNET, subnet and network profile if needed
+    if subnet and vnet_name and not network_profile:
+        network_profile = _get_vnet_network_profile(cmd.cli_ctx, location, resource_group_name, vnet_name, vnet_address_prefix, subnet, subnet_address_prefix)
```

Comment:
If subnet ID is provided, vnet name should not be. I get the feeling this code still should be triggered in that instance?

Reply:
Your correct. Making the change now.